### ISSUE TITLE
Integrate IPP tracking with Isaac sim

### ISSUE THREAD

**Issue by nls5260** [castacks/onr_meta#668]

**Comment by bradygm** [comment]
@nls5260 I noticed this is the only task left in the Integration with Isaac Sim epic. Can you close or update it?

**Comment by nls5260** [comment]
It's blocked by another task. I'll update the Epic with more tasks.
